src/strongbox/ui/jfx.clj

```diff
@@ -1848,27 +1849,36 @@
 
 (defn search-addons-search-field
   [{:keys [fx/context]}]
-  (let [search-state (fx/sub-val context get-in [:app-state :search])]
+  (let [just-x (fn [state]
+                 (select-keys (get-in state [:app-state :search]) [:term :filter-by :page :results-per-page]))
```

Comment:
I like this pattern but it could be condensed to something like:

    (just-in [:path :to :foo [:key1 :key2 :key3]])

and re-used elsewhere